telethon/utils.py

```diff
@@ -1101,104 +1103,85 @@ def resolve_bot_file_id(file_id):
 
     For thumbnails, the photo ID and hash will always be zero.
     """
-    data = _rle_decode(_decode_telegram_base64(file_id))
-    if not data:
+    if not file_id:
         return None
-
-    # This isn't officially documented anywhere, but
-    # we assume the last byte is some kind of "version".
-    data, version = data[:-1], data[-1]
-    if version not in (2, 4):
+    # end if
```

Comment:
Telethon's codebase doesn't really use this style of comments, so I think it's better to remove them.